org.eclipse.iot.fourdiac.Ide.json

```diff
@@ -0,0 +1,62 @@
+{
+    "app-id": "org.eclipse.iot.fourdiac.Ide",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.28",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.openjdk10"
+    ],
+    "command": "/app/4diac-ide/4diac-ide",
+    "modules": [
+        {
+            "name": "openjdk",
+            "buildsystem": "simple",
+            "build-commands": [ "/usr/lib/sdk/openjdk10/install.sh" ]
+        },
+        {
+            "name": "4diac-ide",
+            "buildsystem": "simple",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://download.eclipse.org/4diac/releases/1.9/4diac-ide/4diac-ide-incubation_1.9.1_linux.gtk.x86_64.tar.gz",
+                    "sha256": "e67bccefaa9694f38a0525c866c64e0e2078044c4ba2bf7f2138e4f6214c40a3",
+                    "dest": "4diac-ide"
+                },
+                {
+                    "type": "file",
+                    "path": "4diac-ide.desktop"
+                },
+                {
+                    "type": "file",
+                    "path": "4diac-ide-128.png"
+                },
+                {
+                    "type": "file",
+                    "path": "4diac-ide-256.png"
+                }
+            ],
+            "build-commands": [
+                "mkdir -p /app/share/applications",
+                "mkdir -p /app/share/icons/hicolor/{128x128,256x256}/apps",
+                "install 4diac-ide.desktop /app/share/applications/org.eclipse.iot.fourdiac.Ide.desktop",
+                "for i in 128 256; do install 4diac-ide-$i.png /app/share/icons/hicolor/${i}x${i}/apps/org.eclipse.iot.fourdiac.Ide.png; done",
+                "cp -av 4diac-ide /app",
+                "ln -s /app/jre /app/4diac-ide/jre",
+                "echo '--add-modules=ALL-SYSTEM' >> /app/4diac-ide/4diac-ide.ini"
+            ]
+        }
+    ],
+    "finish-args": [
```

Comment:
on flathub we put the finish args near the top of the file just before the modules section for consistency.